spec/acceptance/class_spec.rb

```diff
@@ -13,4 +13,32 @@ class { 'python': }
       apply_manifest(pp, catch_changes: true)
     end
   end
+  context 'installing python 3' do
```

Comment:
Nit: I always like to separate context blocks
```suggestion

  context 'installing python 3' do
```